### ISSUE TITLE
Training cannot start

### ISSUE THREAD

**Issue by shuuchen** [vacancy/Synchronized-BatchNorm-PyTorch#41]
Hi,

Good job! I tried to used it as 
```
device=torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
model = Model(...)
model = nn.DataParallel(model, device_ids=[0, 1])
model = convert_model(model).to(device)
```

However, it stucked and training couldn 't start.
Have you seem similar problems before ?
